consensus/src/block_storage/block_store.rs

```diff
@@ -342,7 +372,7 @@ impl BlockStore {
             Err(Error::BlockNotFound(parent_block_id)) => {
                 // recover the block tree in executor
                 let blocks_to_reexecute = self
-                    .path_from_root(parent_block_id)
+                    .path_from_commit_root(parent_block_id)
```

Comment:
seems not necessary for commit root? also we probably need to recover the execution phase like this

Reply:
resolved